lib/vtls/gtls.c

```diff
@@ -314,9 +314,12 @@ set_ssl_version_min_max(const char **prioritylist, struct Curl_easy *data)
   long ssl_version = SSL_CONN_CONFIG(version);
   long ssl_version_max = SSL_CONN_CONFIG(version_max);
 
-  if(ssl_version_max == CURL_SSLVERSION_MAX_NONE) {
+  if((ssl_version == CURL_SSLVERSION_DEFAULT) ||
+     (ssl_version == CURL_SSLVERSION_TLSv1))
+    ssl_version = CURL_SSLVERSION_TLSv1_0;
```

Comment:
Isn't there a way to make it less TLS version agnostic? Maybe we upgrade the default TLS version in future and this could avoid some more maintenance

Reply:
That would probably require a much larger take than just fixing this issue for GnuTLS so I rather do that when we find a need to do it.